Gemfile

```diff
@@ -1,48 +1,87 @@
-source ENV['GEM_SOURCE'] || "https://rubygems.org"
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.